features/khost/exec.late

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+K8S_VERSION=${K8S_VERSION:-$(wget --quiet -O- https://storage.googleapis.com/kubernetes-release/release/stable.txt)}
+K8S_VERSION_REPO="${K8S_VERSION%.*}"
+
+wget --quiet -O- "https://pkgs.k8s.io/core:/stable:/${K8S_VERSION_REPO}/deb/Release.key" | gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
```

Comment:
There is no standard for retrieving keys for third party apt repos. 
By downloading and trusting the `Release.key`, we are telling apt to trust everything signed by that key.

If the packages from the apt repo of pkgs.k8s.io are compromised, then we can also not trust the apt key from pkgs.k8s.io. 

I would prefer to have the apt key as a file in a `feature/khost/release.key` file for example.

Reply:
@Vincinator Should have been resolved now